src/plugins/intel_gna/gna2_model_debug_log.cpp

```diff
@@ -369,15 +369,15 @@ void DumpCharArray(std::ostream& dumpFile, const char *carray,  size_t count) {
 
 } // namespace
 
-void DumpGna2Model(const Gna2Model& gnaModel, const std::string dumpFolderNameGNA, bool dumpData) {
+void DumpGna2Model(const Gna2Model& gnaModel, const std::string dumpFolderNameGNA, bool dumpData, const void* memoryBegin) {
```

Comment:
Will we be able to extend this dump after enabling separate memory allocation?

Reply:
Yes, it will be extended